### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by abdielcs**
Added Spanish translation.

### CODE REVIEW

File: Resources/translations/VichUploaderBundle.es.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+download: Descargar
+
+form.label.delete: Eliminar?
```
Comment (javiereguiluz) [inline Resources/translations/VichUploaderBundle.es.yml]: It should be: `¿Eliminar?`

**Comment by abdielcs** [comment]
You're right, I already added the commit.

File: Resources/translations/VichUploaderBundle.es.yml (hunk 1 above)
Comment (abdielcs) [inline Resources/translations/VichUploaderBundle.es.yml]: You're right, I already added the commit.

**Comment by K-Phoen** [comment]
Thanks! :+1:

**Comment by javiereguiluz** [comment]
@abdielcs thanks for the translation!

**Pull request merged**